packages/git/src/browser/git-quick-open-service.ts

```diff
@@ -160,11 +164,11 @@ export class GitQuickOpenService {
     async pull(): Promise<void> {
         const repository = this.getRepository();
         if (repository) {
-            const remotes = await this.getRemotes();
-            const defaultRemote = remotes[0]; // I wish I could use assignment destructuring here. (GH-413)
-            const executeRemote = async (remoteItem: GitQuickOpenItem<string>) => {
+            const [remotes] = await Promise.all([this.getRemotesVerbose('fetch')]);
```

Comment:
Why is the `Promise.all?

Reply:
Sorry was leftover from an earlier version.